src/index.html

```diff
@@ -4,10 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product Card</title>
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Apple iMac 27 Retina 5K Monoblock"
+        >
```

Comment:
Should be only 2 spaces for attributes
```suggestion
      <img
         class="card__image"
         src="images/imac.jpeg"
         alt="Apple iMac 27 Retina 5K Monoblock"
       >
```